src/App.js

```diff
@@ -164,9 +176,9 @@ function App() {
       >
         <h2>Game Over</h2>
         <div className={`result ${result === 'tie' ? 'yellow' : (result === 'o' ? 'red' : 'green')}`}>
-          {result === 'tie' ? "It's a Draw!" : (result === 'o' ? "You LOST!" : "You WON!")}
+          {result === 'tie' ? 'Tie!' : (result === 'o' ? 'You Lose!' : 'You Win!')}
```

Comment:
you want to be careful with the nested ternary conditions here. Nesting ternaries can become very confusing very quickly so you probably want to avoid this in the future. I know it's more compact but it's normally better to go for readability over compactness. I would recommend putting this into a function like this:
```
function displayResult() {
    if (result === 'tie' {
        return <p>Tie!</p>
    } else if (result === 'o') {
        return <p>You LOST!</p>
    } else {
        return <p>You WON!</p>
    }
}
```
Then you can just put `{displayResult}` here and it will do the work for you.